src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -194,21 +204,23 @@ void TimerTick(object sender, EventArgs e)
 
         async Task TimerTick()
         {
-            Debug.Assert(client != null, "TimerTick should not be triggered when there is no IMetricsService");
+            Initialize();
 
-            // Subsequent timer ticks should occur every 8 hours.
-            timer.Interval = TimeSpan.FromHours(8);
+            if (firstRun)
+            {
+                IncrementLaunchCount();
+                timer.Interval = TimeSpan.FromHours(8);
+                firstRun = false;
+            }
 
             if (userSettings == null)
```

Comment:
If you're calling `Initialize` above, then this condition will always be false, right? Do we need any of the code inside this `if` block?